Tribler/community/multichain/database.py

```diff
@@ -286,7 +286,11 @@ def check_database(self, database_version):
         assert int(database_version) >= 0
         database_version = int(database_version)
 
-        if database_version < 1:
+        if database_version < LATEST_DB_VERSION:
+            # Remove all previous data, since we have only been testing so far, and previous blocks might not be
+            # reliable. In the future, we should implement an actual upgrade procedure
+            self.executescript(u"DROP TABLE IF EXISTS multi_chain")
```

Comment:
Can this be done in one method call? Something like (with each statement on a separate line):

```
self.executescript(u"DROP TABLE IF EXISTS multi_chain; DROP TABLE IF EXISTS option;");
```